karg.BLL/Services/FAQs/FAQMappingService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using karg.BLL.DTO.FAQs;
+using karg.BLL.Interfaces.FAQs;
+using karg.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace karg.BLL.Services.FAQs
+{
+    public class FAQMappingService : IFAQMappingService
+    {
+        public async Task<List<AllFAQsDTO>> MapToAllFAQsDTO(List<FAQ> faqs)
+        {
+            var faqsDto = faqs.Select(faq => new AllFAQsDTO
```

Comment:
Have you considered using AutoMapper for this purpose? All these mapping services will be gone, and all the mappings will be in one place - MappingProfile.

Reply:
Sorry, forgot about it, we will definitely use it

Reply:
But there is a certain problem with use, because you still need to loop through the list of animals, for example, to add a list of photos

Reply:
We descoused your request. We thsink use is not relevant at the moment. And wi consider this some latter:)

Reply:
The amount of animals is so little, that iterating them for photos takes negligible performance time. 
Just saying.. you added bunch of services that do what actually mapper does, seems like an overhead and boiler plating.